sdk/dotnet/Pulumi.Automation/Pulumi.Automation.xml

```diff
@@ -1385,21 +1378,6 @@
             <param name="urn">The Pulumi URN to be unprotected.</param>
             <param name="cancellationToken">A cancellation token.</param>
         </member>
-        <member name="M:Pulumi.Automation.WorkspaceStackState.UnprotectAsync(System.Collections.Generic.IEnumerable{System.String})">
-            <summary>
-            Unprotect resources in a stack's state.
-            This command clears the ‘protect’ bit on the provided resource <paramref name="urns"/>, allowing those resources to be deleted.
-            </summary>
-            <param name="urns">The Pulumi URNs to be unprotected.</param>
-        </member>
-        <member name="M:Pulumi.Automation.WorkspaceStackState.UnprotectAsync(System.Collections.Generic.IEnumerable{System.String},System.Threading.CancellationToken)">
-            <summary>
-            Unprotect resources in a stack's state.
-            This command clears the ‘protect’ bit on the provided resource <paramref name="urns"/>, allowing those resources to be deleted.
-            </summary>
-            <param name="urns">The Pulumi URNs to be unprotected.</param>
-            <param name="cancellationToken">A cancellation token.</param>
-        </member>
```

Comment:
Unrelated changes that are made automatically and should have been previously committed.